### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by middaysan**

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -7,8 +7,15 @@ def initialize(user)
     can :read, Project
     can :read, User
     return unless user
-
-    can :manage, Project, user_id: user.id
-    can :manage, Apply, user_id: user.id
+    if user.role.eql?('customer')
```
Comment (middaysan) [inline app/models/ability.rb]: ==

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 module ApplicationHelper
   def show_devise_link?
-    !controller_name.in? %w[sessions registrations passwords]
+    !controller_name.in?(%w[sessions registrations passwords]) &&
```
Comment (middaysan) [inline app/helpers/application_helper.rb]: this

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 module ApplicationHelper
   def show_devise_link?
-    !controller_name.in? %w[sessions registrations passwords]
+    return false if controller_name.in?(%w[sessions registrations passwords])
+
+    return false if action_name == 'edit_role'
```
Comment (RGBD) [inline app/helpers/application_helper.rb]: if controller_name = profile && action_name == 'edit_role'

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -10,8 +10,10 @@
     <h2 class="start-large__subtitle"><%= t(:start_page_subtitle) %></h2>
     <div>
       <% if user_signed_in? %>
-        <%= link_to new_project_path do %>
+        <% if current_user.role == 'customer' %>
```
Comment (RGBD) [inline app/views/home/index.html.erb]: home index and projects index have some common rendering code,
for the future, this templates could be extracted to separate view.

**Pull request merged**